spec/requests/sessions_controller_spec.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Sessions', type: :request do
+  describe 'POST #create' do
+    context 'given a valid e-mail' do
+      it 'returns a successful response with JSON' do
+        application = Fabricate(:application)
+        allow(Doorkeeper::Application).to receive(:first).and_return(application)
```

Comment:
@Tuubz Now I see this stubbing is redundant 😄 We only need to fabricate the Application and that's all. Would you mind to update on other controller_spec 😅

Reply:
Yes, you just need to create an `application` as in your code, it will only use the first one 😄 
```suggestion
       Fabricate(:application)
       # or
       _application = Fabricate(:application)
```

Reply:
@hoangmirs @hoangnguyen92dn Updated 🚀